devops/scripts/register_bundle_with_api.sh

```diff
@@ -14,24 +15,26 @@ function usage() {
 
     Options:
         -r, --acr-name                Azure Container Registry Name
-        -t, --bundle-type             Bundle type, workspace or workspace_service
-        -w, --workspace-service-name  The template name of the user resource
+        -t, --bundle-type             Bundle type: workspace, workspace_service, user_resource or shared_service
+        -w, --workspace-service-name  The template name of the user resource (if registering a user_resource), or shared service (if registering a shared_service)
```

Comment:
I don't think you are using the --workspace-service-name for your shared services. Is the conditional on -d now and not this?

Reply:
Oh yes, a good spot -- forgot to update this message